examples/bot-grnu/pom.xml

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <groupId>com.google.cloud.demo.hangouts.chat.bot</groupId>
+    <artifactId>grnu</artifactId>
+
+    <name>GCP Release Notes Updater</name>
+    <inceptionYear>2019</inceptionYear>
+    <organization>
+        <name>Google Cloud Consulting Services</name>
+        <url>https://cloud.google.com/consulting/</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <!-- AppEngine -->
+        <app.id>[YOUR_PROJECT_ID]</app.id>
```

Comment:
If you change this to ${gcp.project.id} or something and then it could be built with:

    mvn install "-Dgcp.project.id=value"

Reply:
Done with commit "_Update pom.xml parameter. Readme updated_"